lib/src/backends/record_replay.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library file.src.backends.record_replay;
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:file/file.dart';
+import 'package:file/src/io.dart' as io;
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as p;
+
+part 'record_replay/common.dart';
```

Comment:
I would recommend against using parts. They make it very difficult to tell what your internal dependencies look like.